### PR TITLE
feat(slice-machine): disable push changes with connected Git repo

### PR DESCRIPTION
**Pull request by bapmrl**
Completes DT-1901.

### CODE REVIEW

File: packages/slice-machine/src/features/changes/PushChangesButton.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  );
+};
+
+const PushChangesButtonUsingLinkedGitRepos: FC<PushChangesButtonProps> = (
+  props,
+) => {
+  const { linkedGitRepos } = useLinkedGitRepos();
+  return linkedGitRepos.length > 0 ? (
+    <Tooltip
+      content="This repository has been set up to use the Git workflow."
```
Comment (angeloashmore) [inline packages/slice-machine/src/features/changes/PushChangesButton.tsx]: Is there room to add an extra bit of information?

> This project uses the Git workflow. Update Prismic by pushing your changes to ${provider}.

`${provider}` would be "GitHub," "GitLab," etc.

**Review by angeloashmore — CHANGES_REQUESTED** [review]
It works well! I added a wording suggestion, which I will let you and @benjammartin decide.

It looks like E2E tests are broken as of this change, however. Do you have an idea on how to fix it?

File: packages/slice-machine/src/features/changes/PushChangesButton.tsx (hunk 1 above)
Comment (bapmrl) [reply packages/slice-machine/src/features/changes/PushChangesButton.tsx]: I've made the change @angeloashmore 😊

**Review by angeloashmore — APPROVED** [review]
Looks good!

**Comment by bapmrl** [comment]
Thanks @angeloashmore!

**Pull request merged**